runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListView.tsx

```diff
@@ -27,7 +29,30 @@ const UsersListView: FC = () => {
     );
 
     const { activated } = useSelector(usersSelector);
-    const { search, handleSearch, refreshSearch } = useUserSearch(true, limit, page);
+    const { search, handleSearch, refreshSearch: refreshSearchActivated } = useUserSearch(true, limit, page);
+
+    const [userData, setUserData] = useState({});
```

Comment:
empty object is a bad idea. Add type via generics and do not provide anything so it will become undefined under the hood